client/src/modules/stock/requisition/modals/action.modal.html

```diff
@@ -50,6 +56,7 @@
     <div id="depot-supplier" class="form-group">
       <bh-depot-select
         depot-uuid="$ctrl.model.depot_uuid"
+        user="true"
```

Comment:
This appears buggy:

![image](https://user-images.githubusercontent.com/896472/94270885-b77e9880-ff38-11ea-8974-392ecdc64c67.png)